lib/java/com/google/android/material/textfield/IndicatorViewController.java

```diff
@@ -237,6 +237,10 @@ public void onAnimationEnd(Animator animator) {
                   errorView.setText(null);
                 }
               }
+              if (captionViewToShow != null) {
+                captionViewToShow.setTranslationY(0);
```

Comment:
`setTranslationY()` and `setAlpha()` take in floats, so the arguments passed in should be changed from floats to ints.

```suggestion
                captionViewToShow.setTranslationY(0f);
```